src/pages/user/User.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useRef, useEffect } from 'react'
+import { connect } from 'react-redux'
+import { nanoid } from 'nanoid'
+import { Card, Button, Table, Modal, Form, Input, message, Select } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+import { reqUserList, reqCategoryList } from '../../api'
+import { pageSize } from '../../config'
+
+const { Option } = Select;
+function User(props) {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [userList, setUserList] = useState([])
+  const [categoryList, setCategoryList] = useState([])
+  const formInstance = useRef()
+
+  useEffect(() => {
+    //get user list
+    const getUserList = async () => {
+      const result = await reqUserList()
+      const { users } = result.data
+      const newList = users.map((item) => {
+        return {
+          key: item.id,
+          name: item.firstName,
+          email: item.email,
+          number: item.phone,
+          time: "2022-8-11 00:00:01",
+          role: item.company.department
+        }
+      })
+      setUserList(newList)
+    }
+    getUserList()
+
+    //get category list
+    const { categoryInfo } = props
```

Comment:
Consider add custom hook `useCategoryList` to reuse here and in category management page